joomla/joomla-cms
# Jooa11y plugin correction

### Summary of Changes

1. Disable page cache on Jooa11y active state
2. Disable system caching on Jooa11y active state.
3. Load plugin language only if plugin actually runs.
4. Remove useless `$direction` and `$lang` vars.
5. Prettify `isAuthorisedDisplayChecker()` method

### Testing Instructions

Enable Joomla system cache plugin.
Ensure that debug is disabled.
Clear Joomla cache.
Load frontend article as guest.
See cache file in /administrator/cache/page folder
Edit article in backend and perform "Accessibility Check".
See extra file in /administrator/cache/page folder

### Actual result BEFORE applying this Pull Request

An extra cache file is generated for each article

### Expected result AFTER applying this Pull Request

Cache file is not generated.

### Documentation Changes Required

No.
